### PR TITLE
Migrate database to new domain

### PR DESCRIPTION
**Pull request by svenseeberg**
This script updates the following database entries:
* Links in wp_X_posts
* wp_blogs
* wp_x_options

### CODE REVIEW

File: migrate-database.php [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+	/** The Database Collate type. Don't change this if in doubt. */
+	define('DB_COLLATE', $db_configuration['collate']);
+
+	$table_prefix = 'wp_';
+
+	echo "starting migration";
+	define('REPL_DOMAIN_OLD', "vmkrcmar21.informatik.tu-muenchen.de");
+	define('REPL_DOMAIN_NEW', "integreat");
+	define('REPL_PATH_OLD', "/wordpress/");
```
Comment (Ponitac) [inline migrate-database.php]: Did you add this for further Migrations?

Comment (Ponitac) [reply migrate-database.php]: Wouldn't it be better, if we allow the user to define this in the config.ini? Now we have to change the file everytime we use this script

Comment (svenseeberg) [reply migrate-database.php]: Well, the idea is to adopt this script for each run.

Comment (Ponitac) [reply migrate-database.php]: Okay, nevermind then

**Pull request merged**